### PR TITLE
[DO NOT REVIEW] Test: Floating IP integration tests

### PR DESCRIPTION
**Pull request by sodar**
PR adds an integration test for floating IP. Test utilizes the following setup:

- two compute nodes
- two networks, connected with _pass all_ policy
- client container on compute node A
- server container on compute node B
- floating IP is associated with server container's port
- client container pings floating IP

Notes/things to do better:

- `BeforeAll`/`AfterAll` sections are blamelessly copy pasted from `TunnellingWithAgent.Tests.ps1`.
- `Get-Variable` sections in `Before`/`After` sections kinda suck. I do not know how to get rid of them right now.

### CODE REVIEW

**Comment by sodar** [comment]
recheck no bug

File: Test/Utils/ComponentsInstallation.ps1 [hunk 1]
```diff
@@ -80,3 +80,21 @@ function Uninstall-DockerDriver {
     Write-Log "Uninstalling Docker Driver"
     Invoke-MsiExec -Uninstall -Session $Session -Path "C:\Artifacts\docker-driver.msi"
 }
+
+function Install-Components {
```
Comment (m-kostrzewa) [inline Test/Utils/ComponentsInstallation.ps1]: Good move

File: Test/Utils/ContrailUtils.ps1 [hunk 2]
```diff
@@ -142,6 +142,24 @@ function Remove-ContrailVirtualNetwork {
     Invoke-RestMethod -Uri $NetworkUrl -Headers @{"X-Auth-Token" = $AuthToken} -Method Delete | Out-Null
 }
 
+function Get-ContrailVirtualNetworkPorts {
```
Comment (m-kostrzewa) [inline Test/Utils/ContrailUtils.ps1]: Please consider not putting more stuf into ContrailUtils file

Comment (m-kostrzewa) [reply Test/Utils/ContrailUtils.ps1]: IDEA: These seem like implementations of methods in ContrailNetwokManager, so maybe create ContrailNetworkManager subdir, move ContrailnetworkManager.ps1 there, and try to split this ContrailUtils.ps1 file as well? I'm not sure

Comment (m-kostrzewa) [reply Test/Utils/ContrailUtils.ps1]: As discussed IRL, in this PR I have created a subdir : https://github.com/Juniper/contrail-windows-ci/pull/277

Maybe we should add files like "Networks.ps1", "FloatingIP.ps1" etc there

**Comment by krdln** [comment]
Please avoid indirect imports (functions not defined directly in a `.`-imported file):

[shelly](https://github.com/krdln/shelly) output:
<details>
<summary>Click to show</summary>

```
warning: Indirectly imported: Initialize-ComputeServices
  --> ./Test/Tests/Integration/FloatingIP.Tests.ps1
   |
21 |     Initialize-ComputeServices -Session $Session `

warning: Indirectly imported: New-DockerNetwork
  --> ./Test/Tests/Integration/FloatingIP.Tests.ps1
   |
27 |         $NetworkID = New-DockerNetwork -Session $Session `

warning: Indirectly imported: New-Container
   --> ./Test/Tests/Integration/FloatingIP.Tests.ps1
    |
158 |                 New-Container `

warning: Indirectly imported: Remove-AllContainers
   --> ./Test/Tests/Integration/FloatingIP.Tests.ps1
    |
198 |                     Remove-AllContainers -Sessions $Sessions

warning: Indirectly imported: Clear-TestConfiguration
   --> ./Test/Tests/Integration/FloatingIP.Tests.ps1
    |
200 |                     Clear-TestConfiguration -Session $Sessions[0] -SystemConfig $SystemConfig
```
</details>

**Comment by sodar** [comment]
Closed, since #282 and #283 introduce changes from this PR.

**Pull request closed without merging**